styles.css

```diff
@@ -0,0 +1,32 @@
+body{
+    overflow-x: hidden;
```

Comment:
why?

Reply:
to display the div content. 
maybe overflow-x: auto